content/en/get-started/deploy-your-app.md

```diff
@@ -103,150 +114,71 @@ strategies:
    ```
 
-   The CLI returns a `Device Code` and opens your default browser.  Confirm the code in your browser to complete the login process,.
+   The CLI returns a `Device Code` and opens your default browser.  Confirm the code in your browser to complete the login process.
 
-1. Start the deployment from the root of your directory.
+2. Start the deployment from the root of your directory.
 
    ```bash
    armory deploy start  -f deployment.yaml
    ```
 
-   The `deploy start` command starts your deployment. The command returns a **Deployment ID** and a link to your deployment details. 
+   This command starts your deployment, then returns a **Deployment ID** and a link to your deployment details. 
+
+3. Monitor your deployment execution.
 
-1. Monitor your deployment execution.
+   Use the link provided by the CLI to view your deployment in the [CD-as-a-Service Console](https://console.cloud.armory.io/deployments). 
 
-   Use the link provided by the CLI to navigate to your deployment details page in the [CD-as-a-Service Console](https://console.cloud.armory.io/deployments). 
+4. Issue manual approval.
 
-1. Issue manual approval.
+   Once CD-as-a-Service successfully deploys your resources to `staging`, it waits for your manual approval before deploying to `prod`. When the `staging` deployment has completed, click **Approve** to allow the `prod` deployment to begin. 
+   > You must issue manual approvals using the UI. You cannot issue manual approvals using the CLI.
 
-   Since you defined a manual approval to deploy to prod in your strategy, you need to issue your manual approval using the UI. Once the `staging` deployment has completed, click **Approve** to allow the `prod` deployment to begin. Because this is the first time deploying your app, CD-as-a-Service deploys 100% to your prod environment, skipping the defined `trafficSplit` strategy. CD-as-a-Service uses the `trafficSplit` when deploying subsequent versions of your app.
+  Because this is the first time deploying your app, CD-as-a-Service deploys 100% to your prod environment, skipping the defined `trafficSplit` strategy. CD-as-a-Service uses the `trafficSplit` when deploying subsequent versions of your app.
+
+{{% alert title="Important" color="warning" %}}
+CDaaS manages your Kubernetes deployments using ReplicaSets. During the initial deployment of your app, CDaaS deletes the underlying Kubernetes deployment object in a way that leaves behind the ReplicaSet and pods so that there is no actual downtime for your app. These are later deleted when the deployment succeeds.
+
+If your initial deployment fails, you should [manually delete]({{< ref "troubleshooting/tools#initial-deployment-failure-orphaned-replicaset" >}}) the orphaned ReplicaSet.
+{{% /alert %}}
 
 ### Deploy second version
 
 1. Update your Kubernetes manifest to deploy a new version of your app. 
-1. Start the deployment from the root of your directory.
+2. Start the deployment from the root of your directory.
```

Comment:
Numercial lists should all be:
```
1.
1.
1.
```

The list is compiled to the correct numbers. Using all 1s saves you from having to renumber if you insert another step into the top or middle of the list.